Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/exchange/transform/fhir/FHIRTransformTest.java

```diff
@@ -84,13 +90,33 @@ public void testTransformJson() {
         }
     }
 
+    @Test
```

Comment:
Refresher: This test is to ensure that when a new service is added and the service enum list is updated, then there will be the appropriate property added into the .properties file. Otherwise this FHIR transformer breaks and results in a null pointer exception.